addons/web/static/src/core/dialog/dialog.js

```diff
@@ -117,7 +117,10 @@ export class Dialog extends Component {
     }
 
     get contentStyle() {
-        return `top: ${this.position.top}px; left: ${this.position.left}px;`;
+        if (this.props.header) {
```

Comment:
would maybe be more obvious if we had a `this.isMovable` attribute or something.
e.g. in setup:
```js
this.isMovable = this.props.header;
if (this.isMovable) {
    this.position = useState(...);
    useDialogDraggable(...);
    useExternalListener(window, "resize", ...);
}
```
here:
```js
if (this.isMovable) {
    return "top, left";
}
```